typescript/infra/scripts/check/check-validator-version.ts

```diff
@@ -146,8 +147,16 @@ async function main() {
 
         // Get metadata from each storage location
         try {
-          const s3Validator = await S3Validator.fromStorageLocation(location);
-          const metadata = await s3Validator.getMetadata();
+          let validatorInstance;
+          if (location.startsWith('gs://')) {
```

Comment:
might be useful to have this logic somewhere for other consumers to use eventually